docs/guide/rl_tips.rst

```diff
@@ -0,0 +1,235 @@
+Other method, like `TRPO` or `PPO` make use of a *trust region* to minimize that problem by avoiding too large update.
+
+
+How to evaluate an RL algorithm?
+--------------------------------
+
+Because most algorithms use exploration noise during training, you need a separate test environment to evaluate the performance
+of your agent at a given time. It is recommended to periodically evaluate your agent for `n` test episodes (`n` is usually between 5 and 20)
+and average the reward per episode to have a good estimate.
+
+As some policy are stochastic by default (e.g. A2C or PPO), you should also try to set `deterministic=True` when calling the `.predict()` method,
+this frequently leads to better performance.
+Looking at the training curve (episode reward function of the timesteps) is a good proxy but underestimates the agent true performance.
+
+
+We suggest you reading `Deep Reinforcement Learning that Matters <https://arxiv.org/abs/1709.06560>`_ for a good discussion about RL evaluation.
+
+You can also take a look at this `blog post <https://openlab-flowers.inria.fr/t/how-many-random-seeds-should-i-use-statistical-power-analysis-in-deep-reinforcement-learning-experiments/457>`_
+and this `issue <https://github.com/hill-a/stable-baselines/issues/199>`_ by Cédric Colas.
+
+
+Which algorithm should I use?
+=============================
+
+There is no silver bullet in RL, depending on your needs and problem, you may choose one or the other.
+The first distinction comes from your action space, i.e., do you have discrete (e.g. LEFT, RIGHT, ...)
+or continuous actions (ex: go to a certain speed)?
+
+Some algorithms are only tailored for one or the other domain: `DQN` only supports discrete actions, where `SAC` is restricted to continuous actions.
+
+The second difference that will help you choose is whether you can multiprocess or not your training, and how you can do it (with or without MPI?).
+If what matters is the wall clock training time, then you should lean towards `À2C` and its derivates (PPO, ACER, ACKTR, ...).
+Take a look at the `Vectorized Environments <vec_envs.html>`_ to learn more about training with multiple workers.
+
+To sum it up:
+
+Discrete Actions
+----------------
+
+.. note::
+
+	This covers `Discrete`, `MultiDiscrete`, `Binary` and `MultiBinary` spaces
+
+
+Discrete Actions - Single Process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+DQN with extensions (double DQN, prioritized replay, ...) and ACER are the recommended algorithms.
+DQN is usually slower to train (regarding wall clock time) but is the most sample efficient (because of its replay buffer).
+
+Discrete Actions - Multiprocessed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You should give a try to PPO2, A2C and its successors (ACKTR, ACER).
+
+If you can multiprocess the training using MPI, then you should checkout PPO1 and TRPO.
+
+
+Continuous Actions
+------------------
+
+Continuous Actions - Single Process
```

Comment:
I'm a bit hesitant about the single process vs multi process distinction, since if you use `DummyVecEnv` then it's all single-process, but you can still see a performance boost due to concurrency inside of TensorFlow.

Reply:
What I really mean is `VecEnv` support, in the sense that some algorithm are meant to be fast to train (allow `num_envs > 1`) vs sample efficient.